### DISCUSSION TITLE
Footnotes

### DISCUSSION THREAD

**Discussion by thw26** [TriliumNext/Trilium#2044] [Q&A]
Unless I have missed something, I haven't found any discussion or issue on footnotes. Is there a way to do footnotes in a Trilium note?

**Comment by zadam** [comment]
No, not at the moment.

**Reply by thw26** [reply]
Would there be an effective way to load Javascript/jQuery like BigfootJS to accomplish this?

**Reply by zadam** [reply]
No, not really. The problem is that you see notes not in normal webpage but in the HTML editor. Such plugins won't work with the editor (CKEditor).
